src/test/groovy/ee/tuleva/onboarding/auth/ocsp/OCSPServiceSpec.groovy

```diff
@@ -0,0 +1,122 @@
+package ee.tuleva.onboarding.auth.ocsp
+
+import ee.tuleva.onboarding.auth.exception.AuthenticationException
+import org.bouncycastle.asn1.DEROctetString
+import org.bouncycastle.asn1.ocsp.OCSPObjectIdentifiers
+import org.bouncycastle.asn1.ocsp.OCSPResponse
+import org.bouncycastle.asn1.ocsp.OCSPResponseStatus
+import org.bouncycastle.asn1.ocsp.ResponseBytes
+import org.bouncycastle.cert.ocsp.OCSPException
+import org.bouncycastle.cert.ocsp.OCSPReqBuilder
+import org.bouncycastle.cert.ocsp.OCSPResp
+import org.bouncycastle.cert.ocsp.OCSPRespBuilder
+import org.springframework.http.HttpMethod
+import org.springframework.http.ResponseEntity
+import org.springframework.web.client.RestTemplate
+import spock.lang.Specification
+
+import static org.springframework.http.HttpStatus.OK
+
+class OCSPServiceSpec extends Specification {
+    RestTemplate restTemplate = Mock(RestTemplate);
```

Comment:
Use a `TestRestTemplate` instead of mocking it: https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/test/web/client/TestRestTemplate.html